core/lib/src/route/route.rs

```diff
@@ -190,6 +190,8 @@ pub struct Route {
     pub format: Option<MediaType>,
     /// The discovered sentinels.
     pub(crate) sentinels: Vec<Sentry>,
+    /// The doc comment associated with this route.
+    pub(crate) doc_comment: Option<String>,
```

Comment:
```suggestion
    /// The route's docstring, which may be empty.
    pub(crate) docstring: String,
```

Reply:
Once we understand better what we really want to collect from a route, this `docstring: String` field will change to something semantically meaningful, derived from having parsed the docstring as needed as well as having collected information from the guards and responder.